src/OrchardCore.Build/Dependencies.props

```diff
@@ -22,6 +22,7 @@
     <PackageManagement Include="MailKit" Version="2.6.0" />
     <PackageManagement Include="Markdig" Version="0.20.0" />
     <PackageManagement Include="MessagePack" Version="2.1.115" />
+    <PackageManagement Include="Microsoft.Extensions.Configuration.AzureKeyVault" Version="3.1.3" />
```

Comment:
Move it to the `Dependencies.AspNetCore.props` file

    <PackageManagement Include="µ.Extensions.Configuration.AzureKeyVault" Version="$(AspNetCoreVersion)" />

For `AspNetCoreVersion` we currently use `3.1.5`, so we will have to check that it still builds.